starfish/pipeline/filter/gaussian_low_pass.py

```diff
@@ -0,0 +1,49 @@
+from skimage.filters import gaussian
+import numpy
+
+from starfish.munge import swap
+from ._base import FilterAlgorithmBase
+
+
+# todo how do we make these methods available in the API?
+# the starfish import is a bit kludgy because it means we need to:
+# starfish.GaussianFilter.filter()
```

Comment:
agree -- i don't love this API

Reply:
Clarified language. Basically, I'm wondering how our API for function calls within starfish should look, and I think the current version has a bit too much indirection. The above `run` command might help simplify things a little bit.